apps/homescreen/test/unit/everything.me_test.js

```diff
@@ -16,14 +15,16 @@ if (!this.asyncStorage) {
 
 var mocksHelperForEVME = new MocksHelper([
   'LazyLoader',
-  'SettingsListener'
+  'SettingsListener',
+  'L10n'
```

Comment:
Looking at the [MocksHelper](https://github.com/mozilla-b2g/gaia/blob/a14de2c47999a65cff5c06d92625c3ef73e7fb8c/shared/test/unit/mocks/mocks_helper.js#L57) it looks like it only works for objects defined on `window`.  As `mozL10n` is defined on `navigator` I think that you don't need to add it here and instead need to continue doing the `realL10n` juggle.